code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -66,8 +66,13 @@
 					new /obj/effect/hotspot(F)
 	if(iswallturf(T))
 		var/turf/closed/wall/W = T
-		if(prob(reac_volume))
+		//You live by chance, you die by chance
+		if(prob(min(reac_volume,75)))
```

Comment:
why not a log?

Reply:
what?

Reply:
no such thing as tiers if they are both made on the chem machine